src/Security/Authentication/Certificate/src/X509CertificateExtensions.cs

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Linq;
```

Comment:
LINQ? If it's comparing bytes you can use the Span SequenceEqual (https://docs.microsoft.com/en-us/dotnet/api/system.memoryextensions.sequenceequal?view=netcore-3.0#System_MemoryExtensions_SequenceEqual__1_System_ReadOnlySpan___0__System_ReadOnlySpan___0__)

Reply:
Updated to use span sequence equals